src/services/pcn-firewall/datamodel/firewall.yang

```diff
@@ -230,6 +230,26 @@ module firewall {
       }
     }
 
+    action insert {
+      input {
+        leaf id {
+          type uint32;
+        }
+        uses "firewall:rule-fields";
+      }
+      output {
+        leaf id {
```

Comment:
Is this output needed?

Reply:
It is not mandatory, but I think it's nice to have. 
When inserting rules it's nice to have the id referring to the position of the new rule.